src/languages/jsx.coffee

```diff
@@ -10,6 +10,7 @@ module.exports = {
   grammars: [
     "JSX"
     "JavaScript (JSX)"
+    "Babel ES6 JavaScript"
```

Comment:
My only concern is that grammars have the highest priority, over file extension. So by having `Babel ES6 JavaScript` on both files it may incorrectly pick the language. However, Pretty Diff does support both JavaScript and JSX well, I'm not sure about JS-Beautify. So the beautifier could think an ES6 JSX file is actually an ES6 JavaScript file and consequently fail to beautify correctly.
Ideally there would be `Babel ES6 JavaScript (JSX)` (or similar) to indicate a difference in grammar and language between JSX and JavaScript.